packages/scripts/config/webpack.config.js

```diff
@@ -116,6 +116,9 @@ const config = {
 	entry: getWebpackEntryPoints,
 	output: {
 		filename: '[name].js',
+		chunkFileName() {
```

Comment:
```suggestion
		chunkFilename() {
```

See https://webpack.js.org/configuration/output/#outputchunkfilename.

It looks like it can be represented as a string instead of function, too.